cmd/gaia/testnets/README.md

```diff
@@ -88,194 +99,200 @@ Your full node has been cleanly upgraded!
 
 ## Run a Validator Node
 
 [Validators](https://cosmos.network/validators) are responsible for committing new blocks to the blockchain through voting. A validator's stake is slashed if they become unavailable, double sign a transaction, or don't cast their votes. If you only want to run a full node, a VM in the cloud is fine. However, if you are want to become a validator for the Hub's `mainnet`, you should research hardened setups. Please read [Sentry Node Architecture](https://github.com/cosmos/cosmos/blob/master/VALIDATORS_FAQ.md#how-can-validators-protect-themselves-from-denial-of-service-attacks) to protect your node from DDOS and ensure high-availability. Also see the [technical requirements](https://github.com/cosmos/cosmos/blob/master/VALIDATORS_FAQ.md#technical-requirements)). There's also more info on our [website](https://cosmos.network/validators).
 
-Your `pubkey` can be used to create a new validator by staking tokens. You can find your validator pubkey by running:
+Your `cosmosvalpub` can be used to create a new validator by staking tokens. You can find your validator pubkey by running:
 
-```
+```bash
 gaiad tendermint show_validator
 ```
 
 Next, craft your `gaiacli stake create-validator` command:
 
-```
-gaiacli stake create-validator --amount=5steak --pubkey=<your_node_pubkey> --address-validator=<your_address> --moniker=satoshi --chain-id=<name_of_the_testnet_chain> --name=<key_name>
+> _*NOTE:*_  Don't use more `steak` thank you have! You can always get more by using the [Faucet](https://faucetcosmos.network/)!
+
+```bash
+gaiacli stake create-validator \
+  --amount=5steak \
+  --pubkey=$(gaiad tendermint show_validator) \
+  --address-validator=<key_cosmosaccaddr>
+  --moniker=<choose_a_moniker> \
+  --chain-id=gaia-6002 \
+  --name=<key_name>
 ```
 
 You can add more information to the validator, such as`--website`, `--keybase-sig`, or `--details`. Here's how:
 
-```
-gaiacli stake edit-validator --details="To the cosmos !" --website="https://cosmos.network"
+```bash
+gaiacli stake edit-validator \
+  --details="To the cosmos!" \
+  --website="https://cosmos.network"
 ```
 
 View the validator's information with this command:
 
-```
-gaiacli stake validator --address-validator=<your_address> --chain-id=<name_of_the_testnet_chain>
+```bash
+gaiacli stake validator \
+  --address-validator=<your_address> \
+  --chain-id=gaia-6002
 ```
 
-To check that the validator is active, look for it here:
+Your validator is active if the following command returns anything:
 
+```bash
+gaiacli advanced tendermint validator-set | grep "$(gaiad tendermint show_validator)"
 ```
-gaiacli advanced tendermint validator-set
-```
 
-**Note:** To be in the validator set, you need to have more total voting power than the 100th validator.
+You should also be able to see your validator on the [Explorer](https://explorecosmos.network/validators). You are looking for the `bech32` encoded `address` in the `~/.gaiad/config/priv_validator.json` file.
```

Comment:
this is the first mention of `bech32` in the tutorial. would be cool to mention it in the section way up top where you list the 3 types of keys.